hw/market.py

```diff
@@ -9,20 +15,30 @@ def has_drink_with_title(self, title=None) -> bool:
         :param title:
         :return: True|False
         """
-        pass
+        if (title in self.wines) or (title in self.beers):
+            return True
+        else:
+            return False
```

Comment:
в методах проверки можно сразу так делать
```suggestion
        return (title in self.wines) or (title in self.beers)
```

Reply:
Понял. Исправил.